client/lib/signup/step-actions/index.js

```diff
@@ -437,38 +437,32 @@ export async function setStoreFeatures( callback, { siteSlug }, stepProvidedItem
 			<!-- /wp:column -->
 
 			<!-- wp:column {"style":{"spacing":{"padding":{"right":"40px","left":"40px"}}}} -->
-			<div class="wp-block-column" style="padding-right:40px;padding-left:40px"><!-- wp:heading {"level":1,"style":{"typography":{"fontSize":"40px"},"spacing":{"margin":{"top":"0px"}}}} -->
-			<h1 id="item-name" style="font-size:40px;margin-top:0px">${ translate( 'Item Name' ) }</h1>
+			<div class="wp-block-column" style="padding-right:40px;padding-left:40px"><!-- wp:heading {"level":1,"placeholder":"Item name","style":{"typography":{"fontSize":"40px"},"spacing":{"margin":{"top":"0px"}}}} -->
```

Comment:
shouldn't this still be wrapped in `translate`?

Reply:
Oh, if the content is fetched from the server, instead of hardcoded, we should find out if there was a way to add translated placeholders . . .

Reply:
With #61160 merged and looking good, I think we can close this. 👍